Recruit-Training/Basic-Recruit-Training/person/Person.cpp

```diff
@@ -0,0 +1,57 @@
+#include "Person.h"
+#include <iostream>
+
+using namespace::std;
```

Comment:
Not sure if you can do that.... but either way if you're going to use it do it like `using namespace std`. Also it is bad practice to use it in general so I wouldn't recommending using it.